Cdk support

jyriseiger opened the pull request:
Added first version of CDK support.

Commit:
Adding CDK support.

Commit:
Finalized support for CDK.

Commit:
Added new version of demo.gif and script for generating it.

Maurits-de-Groot commented on demo.tape:
Maybe we should NOT add this file?

jyriseiger replied:
agree, removed it.

Commit:
Removed unneeded file.

Maurits-de-Groot commented on susscanner/cli.py:
Could we use subprocess.Popen over system.os?

Maurits-de-Groot commented on README.md:
Under Prerequisites we should list cdk as optional pre-req for the cdk support

Maurits-de-Groot replied:
Looks good, thanks for adding

Commit:
Used run_command to execute cdk command.

Commit:
Added CDK prerequisite.

Maurits-de-Groot committed:
Update cli.py - run_command(command: str)

for subprocess to work in Windows we need to provide the full path of the command - not just the command (i.e., cdk or cfn-guard)

[code]
+ # Get the full path of the command to ensure execution in Windows
+args[0] = shutil.which(args[0])
